src/riak_kv_get_fsm.erl

```diff
@@ -180,23 +180,35 @@ init({test, Args, StateProps}) ->
     {ok, validate, TestStateData, 0}.
 
 %% @private
-prepare(timeout, StateData=#state{bkey=BKey={Bucket,_Key},
+prepare(timeout, StateData=#state{bkey = {Bucket, PossiblyTSKey},
```

Comment:
Actually, scratch that for a better idea...

Match on Key, and anywhere we need a partition key, we call a `?PARTITION_KEY` macro, also a `?LOCAL_KEY` macro, and the logic of divvying up the tuple goes into those macros.

This way the get/put/delete paths can remain virtually untouched but we're being very explicit whenever we call into another function about which semantic purpose is relevant.